src/Router.ts

```diff
@@ -70,13 +70,6 @@ export default class Router implements IRouter {
     * code hinting / auto-completion.
     */
    public addSubRouter(path: PathParams, router: Router): this {
-      // Note: this overriding of the case sensitivity of the passed-in router is likely
-      // ineffective for most usecases because the user probably created their router and
-      // added a bunch of routes to it before adding that router to this one. When the
-      // routes are created (technically the `IRequestMatchingProcessorChain` objects, in
-      // particular `RouteMatchingProcessorChain`), the case sensitivity is already set
-      // (inherited from the router that created the chain).
-      router.routerOptions.caseSensitive = this.routerOptions.caseSensitive;
```

Comment:
we need to test what happens when a router is added to an app

Reply:
I verified this using the following test case:

```javascript
const express = require('express'),
      port = 3000,
      app = express();

// Test request: GET /World
// The 'case sensitive routing' application setting is NOT passed on to routers
// registered with app.use

const router = express.Router();

app.set('case sensitive routing', true);
app.use(router); // set this BEFORE adding any routes

router.get('/World', (req, resp, next) => {
   console.log('uppercase /World');
   console.log('subApp is case sens?', subApp.get('case sensitive routing'));
   next();
});
router.get('/world', (req, resp, next) => {
   console.log('lowercase /world');
   console.log('subApp is case sens?', subApp.get('case sensitive routing'));
   next();
});
router.get('*', (req, resp) => { resp.send('world'); });

app.listen(port, () => console.log(`Example app listening on port ${port}!`));
```